acceptance-tests/src/test/resources/blackbox/application-mem-auth.yml

```diff
@@ -0,0 +1,36 @@
+#SPDX-FileCopyrightText: 2017-2024 Enedis
+#SPDX-License-Identifier: Apache-2.0
+
+chutney:
+  security:
+    users:
+      -
+        id: robert
+        name: robert
+        firstname: robert
+        lastname: robert
+        mail: robert@robert.com
+        password: $2a$10$yGDGbzWT7Xf6y8ifVx4KGumLH6cHObg841wNq8/EsNuHCCg/NuK4G
```

Comment:
We can maybe add here the password in plain in comment to not forget it, even if we can find it in tests.